src/screens/AffiliationViews/DeploymentView/DetailsView/VersionView/containers/RedeployRowAndVersionTable/RedeployRowAndVersionTable.tsx

```diff
@@ -20,13 +22,58 @@ export const RedeployRowAndVersionTable: React.FC<Props> = ({
   isDeploying,
   releaseTo,
   isFetchingConfiguredVersionTag,
+  auroraConfigFiles,
+  affiliation,
 }) => {
   const [versionBeingDeploy, setVersionBeingDeploy] =
     useState<string | undefined>();
 
+  function changeVersionInFile(
+    fileName: string,
+    fileContent: string,
+    version: string
+  ) {
+    if (fileName.endsWith('json')) {
+      const parsedApplicationFile: JSON = JSON.parse(fileContent);
+      return JSON.stringify(
+        { ...parsedApplicationFile, version: version },
+        null,
+        2
+      );
+    } else if (fileName.endsWith('yaml')) {
+      const parsedApplicationFile = YAML.parseDocument(fileContent);
+      parsedApplicationFile.set('version', version);
+      return parsedApplicationFile.toString();
+    } else {
+      console.log('filetype used for application file is not supported');
```

Comment:
To ting:
* Er det kun yaml og json som er lov? er f.eks yml lov?
* Dersom boober vil stoppe filer som ikke slutter på .yaml og .json så trengs kanskje ikke else her å håndteres?

Reply:
Ja du kan kanskje sjekk om det er json hvis ikke så kan vi anta at det er yaml. Hvis innholdet ikke er yaml så vil parseren feile, her bør du også ha en try catch for å håndtere de feilene.